build.gradle.kts

```diff
@@ -8,4 +8,11 @@ plugins {
 
 subprojects {
   group = "org.hypertrace.core.graphql"
+
+  pluginManager.withPlugin("java") {
```

Comment:
would be a good candidate for rolling into one of the gradle plugins, especially since that's where the runtime version is defined.

Reply:
agreed!